src/page/renderer/field.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
```

Comment:
again :P filename field doesn't say anything specific to me. Maybe it's a field of rice? 🍚 😄

Reply:
resolved on private conversation